test/config_test/example_configs_test.cc

```diff
@@ -20,10 +34,9 @@ TEST(ExampleConfigsTest, All) {
   RELEASE_ASSERT(::chdir(directory.c_str()) == 0, "");
 
 #if defined(__APPLE__) || defined(WIN32)
-  // freebind/freebind.yaml is not supported on macOS or Windows and is disabled via Bazel.
-  EXPECT_EQ(37UL, ConfigTest::run(directory));
+  EXPECT_EQ(config_file_count - LinuxOnlyConfigFileTestCount, ConfigTest::run(directory));
```

Comment:
not sure if both of these vars should be lower case etc

Reply:
This `LinuxOnlyConfigFileTestCount` is a constant, we use PascalCase or SCREAMING_SNAKE_CASE interchangeably (up to you, I think we haven't consistent in constant naming convention, this is one of the deviation from Google's C++ style-guide https://google.github.io/styleguide/cppguide.html#Constant_Names).

Reply:
k - ill leave it as is unless you suggest to change it